### PR TITLE
add streams info and publisher display to publish event sent to event handlers (videoroom)

### PR DESCRIPTION
**Pull request by aberrios1**
Pretty much my first open source contribution anywhere, be nice :smile: 

Added info about streams published on publish event sent to event handlers.

Wanted to get the stream descriptions in our publish events.

Let me know if this is useful or not, or if something needs changing.

### CODE REVIEW

**Comment by januscla** [comment]
Thanks for your contribution, @aberrios1! Please make sure you sign our [CLA](https://cla.conf.meetecho.com/janus), as it's a required step before we can merge this.

**Comment by lminiero** [comment]
Sorry for the late response @aberrios1 and thanks for your first contribution! I'll have a look later today and post a review.

File: plugins/janus_videoroom.c [hunk 1]
```diff
@@ -3595,7 +3595,37 @@ static void janus_videoroom_notify_about_publisher(janus_videoroom_publisher *p,
...
+				} else if(ps->type == JANUS_VIDEOROOM_MEDIA_VIDEO) {
+					json_object_set_new(mediainfo, "codec", json_string(janus_videocodec_name(ps->vcodec)));					
+					if(ps->muted)
+						json_object_set_new(mediainfo, "moderated", json_true());
+					if(ps->simulcast)
+						json_object_set_new(mediainfo, "simulcast", json_true());
+					if(ps->svc)
+						json_object_set_new(mediainfo, "svc", json_true());
+				}
+			}
```
Comment (lminiero) [inline plugins/janus_videoroom.c:3624]: Why no datachannel?

Comment (lminiero) [reply plugins/janus_videoroom.c:3624]: Ah nevermind, this is similar to how we advertise streams to subscribers anyway: datachannels are notified as part of the `type`, and need no extra info as audio and video do instead.

**Comment by lminiero** [comment]
Looks good to me, thanks, merging! :+1:

**Pull request merged**